pkg/controller/atlasdatabaseuser/atlasdatabaseuser_controller.go

```diff
@@ -282,28 +286,32 @@ func (r *AtlasDatabaseUserReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
-func managedByAtlas(ctx context.Context, atlasClient *mongodbatlas.Client, projectID string, log *zap.SugaredLogger) customresource.AtlasChecker {
+func managedByAtlas(ctx context.Context, dus dbuser.AtlasUsersService, projectID string, log *zap.SugaredLogger) customresource.AtlasChecker {
```

Comment:
Shouldn't we wait for the removal of the ownership detection code to be merged?

Reply:
rebased